docs/administrator/configuration/configure-controllers.md

```diff
@@ -116,6 +116,47 @@ More details please refer to:
 - [ttl after finished controller](https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#ttl-after-finished-controller)
 - [clean up finished jobs automatically](https://kubernetes.io/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically)
 
+#### bootstrapsigner
+
+The `bootstrapsigner` controller runs as part of `kube-controller-manager`.
+The tokens are also used to create a signature for a specific ConfigMap used in a "discovery" process through a `bootstrapsigner` controller
+
+For the Karmada control plane, we also provide `cluster-info` ConfigMap in `kube-public` namespace. This is used early in a cluster bootstrap process before the client trusts the API server. The signed ConfigMap can be authenticated by the shared token.
+
+> Note: this controller currently is used to register member clusters with PULL mode by `karmadactl register`.
+
+More details please refer to:
+- [bootstrap tokens overview](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/#bootstrap-tokens-overview)
+- [configmap signing](https://kubernetes.io/docs/reference/access-authn-authz/bootstrap-tokens/#configmap-signing)
+
+#### tokencleaner
+
+The `tokencleaner` controller runs as part of `kube-controller-manager`.
+Expired tokens can be deleted automatically by enabling the tokencleaner controller on the controller manager.
+
+> Note: this controller currently is used to register member clusters with PULL mode by `karmadactl register`.
```

Comment:
I like the idea that telling people how we use it in detail, thus users can easily choose if they need to enable it.